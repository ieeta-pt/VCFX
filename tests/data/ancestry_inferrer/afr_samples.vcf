##fileformat=VCFv4.2
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	AFR_SAMPLE1	AFR_SAMPLE2	AFR_SAMPLE3
1	100	rs1	A	G	30	PASS	AF=0.10	GT	0/0	0/1	0/0
1	200	rs2	C	T	40	PASS	AF=0.70	GT	1/1	1/0	1/1
1	300	rs3	G	A	50	PASS	AF=0.15	GT	0/0	0/1	0/0
2	250	rs6	C	G	80	PASS	AF=0.60	GT	1/0	1/1	1/0
2	350	rs7	G	C	90	PASS	AF=0.85	GT	1/1	1/1	1/0
