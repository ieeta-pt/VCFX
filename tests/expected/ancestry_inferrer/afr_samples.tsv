Sample	Inferred_Population
AFR_SAMPLE1	AFR
AFR_SAMPLE2	AFR
AFR_SAMPLE3	AFR
